packages/blade/src/tokens/global/breakpoints.ts

```diff
@@ -1,25 +1,69 @@
 export type Breakpoints = Readonly<{
-  /** max width: 320px  */
+  /**
+   * `base` is used for base styling. It starts from 0px till the next existing token
+   *
+   * Think of this as styles without any media query.
+   *
+   * ### Example
+   *
+   * This code will set margin as `spacing.2` on "m" size screens and above. And as `spacing.1` on less than "m" size screens
+   * ```jsx
+   * <Box margin={{ base: 'spacing.1', m: 'spacing.2' }} />
+   * ```
+   *
+   * This roughly translates into -
+   *
+   * ```js
+   * styled.div(() => ({
+   *   margin: 'spacing.1',
+   *   [`@media screen and (min-width: 768px)`]: {
+   *     Box: {
+   *       margin: 'spacing.2'
+   *     }
+   *   }
+   * }))
+   * ```
+   */
+  base: number;
+  /**
+   * `@media screen and (min-width: 320px)`
+   *
+   * Small Mobiles
+   */
   xs: number;
-  /** min width: 321px and max width: 480px */
+  /**
+   * `@media screen and (min-width: 480px)`
+   *
+   * Mobiles and Small Tablets
+   */
   s: number;
-  /** min width: 481px and max width: 768px */
+  /**
+   * `@media screen and (min-width: 768px)`
+   *
+   * Medium and Large Tablets. Would be ideal to use this as breakpoint while writing desktop css
```

Comment:
> Would be ideal to use this as breakpoint while writing desktop css

This part is making it a bit confusing because initially it mentions medium and large tablets. Perhaps you can make the intent clearer - eg. dimentions above `m` can be treated as desktop in mobile-first approach (with min-width)

Reply:
done